features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitor.kt

```diff
@@ -730,6 +739,13 @@ internal class DatadogRumMonitor(
         sdkCore.internalLogger.reportFeatureOperationApiUsage(ActionType.FAIL)
     }
 
+    @ExperimentalRumApi
+    override fun reportAppFullyDisplayed() {
```

Comment:
probably this should be moved? otherwise it is in the `// region Feature Operations`